README.md

```diff
@@ -1,95 +1,94 @@
 # AtlasSupportSDK
 
-**AtlasSupportSDK** is a mobile SDK that integrates a real-time chat widget into iOS applications. 
+[![Version](https://img.shields.io/cocoapods/v/AtlasSupportSDK.svg?style=flat)](https://cocoapods.org/pods/AtlasSupportSDK)
+[![License](https://img.shields.io/cocoapods/l/AtlasSupportSDK.svg?style=flat)](https://cocoapods.org/pods/AtlasSupportSDK)
+[![Platform](https://img.shields.io/cocoapods/p/AtlasSupportSDK.svg?style=flat)](https://cocoapods.org/pods/AtlasSupportSDK)
 
----
+
+**AtlasSupportSDK** is a mobile SDK that integrates a real-time chat widget into iOS applications. 
 
 ## Installation
 
+You can add Swift SDK to your application in two ways: via **SPM** or **CocoaPods**.
+
 ### SPM
 
-1. Open Xcode & Add the SDK
-2. Open your Xcode project.
-3. Go to File → Add Packages.
-4. In the search bar, enter the repository URL of the SDK:
+1. In the Xcode, when your project is opened, select **File → Add Package Dependencies...**
+
+2. In the opened window enter URL of the Atlas SDK in the search bar:
 
 ```
 https://github.com/atlas-support-inc/mobile-sdk-ios
 ```
 
-5. Select the SDK package from the list.
-6. Click Add Package. Do not forget "Add to Target".
-7. Choose Dependency Rule (e.g., "Up to Next Major Version" or "Exact Version").
+3. Select the **AtlasSupportSDK** from the search results, configure **Dependency Rule** and **Add to Project** settings, and click **Add Package** button.
```

Comment:
> configure **Dependency Rule** and **Add to Project** settings

What is **Add to Project** settings? I could not find it.

Also, this could be sublist
3. Select the **AtlasSupportSDK** from the search results.
    - Configure **Dependency Rule** (e.g., "Up to Next Major Version" or "Exact Version").
    - Add SDK to  "YourTarget"
    - Finally click the **Add Package** button.